modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -5243,11 +5243,13 @@ draft-version=Draft Version
 draft-x=Draft &#149; {0}
 drafts=Drafts
 drag-and-drop-criterion-from-the-right-to-add-rules=Drag and drop criterion from the right to add rules.
-drag-and-drop-or-click-to-upload=Drag & Drop or Click to Upload
 drag-and-drop-over-an-existing-criteria-to-form-groups=Drag and drop over an existing criteria to form groups.
 drag-and-drop-to-upload=Drag and Drop to Upload
 drag-and-drop-to-upload-or-x=<strong>Drag</strong> & <strong>Drop</strong> to <strong>Upload</strong> or {0}
 drag-and-drop-to-upload-to-x-or=<strong>Drag</strong> & <strong>Drop</strong> to <strong>Upload</strong> to {0} or
+drag-and-drop-your-files-or-click-to-upload=Drag & Drop your files or Click here to Upload
+drag-and-drop-your-images-or-click-to-upload=Drag & Drop your images or Click here to Upload
+drag-and-drop-your-videos-or-click-to-upload=Drag & Drop your videos or Click here to Upload
```

Comment:
Capitalization is wrong. Use https://capitalizemytitle.com/ to fix it.

Reply:
I directly copy the design text, I will fix it